vent/controller/control_module.py

```diff
@@ -397,8 +397,12 @@ def __test_for_alarms(self):
                                   value=self._DATA_PRESSURE)
             if time.time() - self.HAPA.start_time > self.cough_duration:       # 100 ms active to avoid being triggered by coughs
                 self.__SET_PIP = 30                 # Default: PIP to 30
-                self.__control_signal_out = np.inf
-                self.__control_signal_in  = 0
+                for i in range(5):                   # Make sure to send this command for 100ms -> release pressure immediately
```

Comment:
this is new. Sometimes a valve might be stuck, so the first thing is making sure it is open, by sending open command multiple times